.vscode/c_cpp_properties.json

```diff
@@ -3,19 +3,32 @@
         {
             "name": "Mac",
             "includePath": [
-                "${workspaceFolder}/**"
```

Comment:
I think it may be better to put `.vscode` into `.gitignore` as paths here are specific to each computer.